Models/Device.js

```diff
@@ -8,12 +8,19 @@ const deviceKey = 'DeviceKey'
 export default class DeviceModel {
 
     /**
-     * Get the info for the current device.
+     * Get the info for the current device. Or call saveDeviceInfo()
+     * if there is no info saved to this device.
      */
     static async getDeviceInfo() {
         return AsyncStorage.getItem(deviceKey)
```

Comment:
When we use the async keyword, there is no need for chaining .then()'s

We could change it to this:

```
const deviceInfo = await AsyncStorage.getItem(deviceKey)

if (deviceInfo) {
  return deviceInfo
}

// No info found, lets save the current info and return it
await DeviceModel.saveDeviceInfo()
return await AsyncStorage.getItem(deviceKey)
```